lib/interface_quda.cpp

```diff
@@ -4940,13 +4939,11 @@ void performTwoLinkGaussianSmearNStep(void *h_in, QudaQuarkSmearParam *smear_par
   const QudaParity  parity   = QUDA_INVALID_PARITY;
   for (int i = 0; i < smear_param->n_steps; i++) {
     if (i > 0) std::swap(in, out);
-    blas::ax(ftmp, in);
-    blas::axpy(a, in, temp1);
 
     qsmear_op.Expose()->SmearOp(out, in, a, 0.0, smear_param->t0, parity);
-    logQuda(QUDA_DEBUG_VERBOSE, "Step %d, vector norm %e\n", i, blas::norm2(out));
-    blas::xpay(temp1, -1.0, out);
-    blas::zero(temp1);
+    if (getVerbosity() >= QUDA_DEBUG_VERBOSE)
```

Comment:
No need for this if statement, the `logQuda` function wraps this in there

Reply:
Thanks for pointing this out -- `logQuda` is a more clever macro than I realized.  I've pushed a commit to remove this redundant `if` statement